build.gradle.kts

```
@@ -79,6 +81,30 @@ subprojects {
                 if (file.length() == 0L) throw IllegalStateException("Distribution file: $distributionFilePath is empty")
             }
         }
+
+        apply<ReleasePlugin>()
+
+        configure<PublishExtension> {
+            userOrg = Config.Sentry.userOrg
+            groupId = project.group.toString()
+            publishVersion = project.version.toString()
+            desc = Config.Sentry.description
+            website = Config.Sentry.website
+            repoName = if (project.name.contains("android")) Config.Sentry.androidRepoName else Config.Sentry.javaRepoName
```

The suggestion is this, though PR is good to merge:

```suggestion
            repoName = if (project.name.contains("android")) Config.Sentry.androidBintrayRepoName else Config.Sentry.javaBintrayRepoName
```

---

it makes easier to read but also verbose, its setting in the bintray plugin `repoName` field so it's kinda implicit that its bintray's repo name, I'm fine with this change though.
If you had a doubt about the var you are probably right, I wrote this part myself :D